ticket/constants.py

```diff
@@ -0,0 +1,8 @@
+CREATED_STATE = "CREATED"
+CANCELLED_STATE = "CANCELLED"
+FULFILLED_STATE = "FULFILLED"
+ORDER_STATES = (
+    (CREATED_STATE, "Created"),
+    (CANCELLED_STATE, "Cancelled"),
+    (FULFILLED_STATE, "Fulfilled"),
+)
```

Comment:
This would have been TextChoice class but I was limited by the low version of Django in the requirements file.